content/en/cloudcraft/getting-started/connect-amazon-eks-cluster-with-cloudcraft.md

```diff
@@ -13,6 +13,9 @@ To learn more about RBAC configuration and IAM entities, see [Managing users or
 
 ## Prerequisites
 
+<div class="alert alert-info">Scanning EKS clusters requires public endpoints. Read <a href="https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html">Amazon EKS cluster endpoint access control
```

Comment:
```suggestion
<div class="alert alert-info">Scanning EKS clusters requires public endpoints. For more information, see <a href="https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html">Amazon EKS cluster endpoint access control.
```

Reply:
Thanks for the feedback! I've pushed a change which incorporates your suggestion, but which also removes a part of the sentence which was left hanging and wouldn't have made sense if it went with your change.
If you're happy with this revision so am I!